docs/coverage/repositories/repository-dashboard.md

```diff
@@ -0,0 +1,79 @@
+---
+product: coverage
+---
+
+# [Coverage] Repository Dashboard
+<!--TODO
+    Review page, see https://codacy.atlassian.net/browse/COV-115 for more details-->
+
+The **Repository Dashboard** provides an overview of the repository coverage and items that require your attention.
+
+To access your Repository Dashboard, select a repository from the [Organization Dashboard](../../organizations/organization-overview.md) or open a repository on any other page and select **Dashboard** on the left navigation sidebar.<!--TODO Review navigation-->
+
+!!! tip
+    You can share the URL of the Repository Dashboard for your **public repositories** to allow other people to see your repository coverage metrics, even if they aren't registered on Codacy.
+
+![Repository Dashboard](images/repository-dashboard.png)<!--TODO Update-->
+
+The top of the Repository Dashboard displays:
+
+-   The name of the repository
+-   A drop-down list that selects which branch of your repository to display on the dashboard
+
+On the Repository Dashboard you have the following areas to help you monitor your repository:
+
+-   [Quality evolution chart](#quality-evolution-chart)
+-   [Coverage](#coverage)
+-   [Open pull requests](#open-pull-requests)
+
+The following sections provide a detailed overview of each dashboard area.
+
+## Quality evolution chart
+
+The **Quality evolution** chart displays the evolution of the repository code quality regarding [code coverage](../../faq/code-analysis/which-metrics-does-codacy-calculate.md#code-coverage). Click on **Last 3 months**, **Last 31 days**, or **Last 7 days** to select the time interval of the historical data to display on the chart.
+
+The tab displays the following information:
+
+-   A green or red indicator depending if coverage is within the acceptable level or not
+-   The current coverage value
+-   The coverage variation introduced by the last commit
+
+!!! note
+    The chart only displays a value if Codacy received coverage data for the most recent commit. This is because one commit can <span class="skip-vale">easily</span> change the size or number of files on the repository, or even remove some files that had coverage information.
+
+The chart also displays the **trendline** based on the past behavior and the **quality goal** defined on the repository [quality settings](../../repositories-configure/adjusting-quality-settings.md).
+
+![Quality evolution chart](images/repository-dashboard-quality-evolution.png)<!--TODO Update-->
```

Comment:
Another inverted image, where the original page and the Coverage page point to each other's image in the built files (I see no changes in the source Markdown of the original page).

Seems like a build process issue tied to the relative paths of the assets.

Could this be an mkdocs bug?

Reply:
Another great catch!

I had updated the wrong screenshot files earlier on in this commit: https://github.com/codacy/docs/pull/1658/commits/0e3a00792ab0e3c59a116fe4f79f872f4d034d59

I already reverted the changes to the original screenshots and updated the correct ones: https://github.com/codacy/docs/pull/1658/commits/efebfe243416df5ce1fa11a434a2411ca5a13092